### PR TITLE
refs task/5822 Comments Manage

### PR DESCRIPTION
**Pull request by NamHuy18**
Quản lý bình luận
https://edu-redmine.framgia.vn/issues/5822

### CODE REVIEW

**Comment by NamHuy18** [comment]
ready

File: resources/views/admin/comment.blade.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                                    <span>{{ trans('setting.deleteAll') }}</span>
+                                </a>
+                            </div>
+                        </td>
+                    </tr>
+                </tfoot>
+                <tbody>
+                    @foreach ($comment as $cm)
+                        <tr class='cat_{{$cm->id}}'>
+                        <td><input type="checkbox" class="check-product" name="id[]" value="{{$cm->id}}" /></td>
```
Comment (BunnyPi04) [inline resources/views/admin/comment.blade.php]: convention

File: routes/web.php [hunk 2]
```diff
@@ -74,6 +74,11 @@
 	    Route::post('edit/{id}', 'BannerController@postEdit');
 	    Route::get('delete/{id}', 'BannerController@getDelete')->name('deleteBanner');
 	});
+	Route::group(['prefix'=>'comment'], function()
```
Comment (BunnyPi04) [inline routes/web.php]: convention

**Comment by NamHuy18** [comment]
ready

**Pull request merged**